### ISSUE TITLE
Error, 'Quart' object has no attribute 'session_cookie_name'

### ISSUE THREAD

**Issue by Meteord** [kroketio/quart-session#15]
Hello,

i'm using Quart=**0.19.3** and quart-session=**2.0.0**. After starting the following minimal sample
``` python
from quart import Quart, session
from quart_session import Session

app = Quart(__name__)

app.config['SESSION_TYPE'] = 'memcached'

Session(app)

@app.route('/')
async def hello():
    session["foo"] = "bar"
    return "session key 'foo' set"

@app.route('/foo')
async def foo():
    return session.get("foo", "session key 'foo' not found")
app.run()
```

I'm getting the following error:
```
quart_session\sessions.py", line 95, in open_session
    sid = request.cookies.get(app.session_cookie_name)
AttributeError: 'Quart' object has no attribute 'session_cookie_name'
``` 

Thanks for your time and your help!

**Comment by butchewing** [comment]
I'm getting the same error after updating dependancies. I tried downgrading Quart and/or Quart-Session but kept getting other related errors. I did [see this](https://stackoverflow.com/questions/76139355/flask-object-has-no-attribute-session-cookie-name) which may be of some help.

**Comment by kroketio** [comment]
Thanks, looks like Flask changed some things which ultimately influenced Quart. Fixed in version `2.1.0`
